CMake/functions/emscripten_system_library.cmake

```diff
@@ -0,0 +1,18 @@
+# This function defines a target that points to an Emscripten system library.
+#
+# Arguments:
+#   LIB_NAME: a human-readable library name.
+#   TARGET_NAME: the library target name
+#   ...ARGN: Emscript flags.
+#
+# Example:
+#   emscripten_system_library("SDL2_image" SDL2::SDL2_image USE_SDL_IMAGE=2 "SDL2_IMAGE_FORMATS='[\"png\"]'")
+function(emscripten_system_library LIB_NAME TARGET_NAME)
+  set(_lib_target "EMSCRIPTEN_${LIB_NAME}")

```

Comment:
This variable doesn't appear to be used.

Reply:
Fixed